app/helpers/arf_reports_helper.rb

```diff
@@ -22,7 +22,7 @@ def result_tag(level)
           else
             "warning"
           end
-    "class='label label-#{tag}'".html_safe
+    "class='label label-#{tag} result-filter-tag'".html_safe
```

Comment:
This is all we need from this PR now after https://github.com/theforeman/foreman/pull/4728 was merged. I already tested it works, please remove the rest.